dev/MRTCore/TestApps/winui_desktop_packaged_app/winui_desktop_packaged_app (Package)/build/Microsoft.WinUI.AppX.targets

```diff
@@ -0,0 +1,174 @@
+<!--
+  Copyright (c) Microsoft Corporation. Licensed under the MIT License
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    
+    <!-- Replaces AppxManifest WinRT registrations for Microsoft.UI.Xaml.dll, post-GenerateAppxManifest. -->
+    <!-- Target runs before manifest validation occurs. Requires output file to support incremental builds. -->
+    <Target Name="_ReplaceMUXWinRTRegistrations" Condition="'$(Platform)' != 'ARM64' AND '$(Platform)' != 'ARM'" BeforeTargets="_CreateFinalAppxManifestItem" AfterTargets="_GenerateCurrentProjectAppxManifest" Inputs="$(MSBuildAllProjects);@(_GenerateCurrentProjectAppxManifestInput)" Outputs="$(FinalAppxManifestName).incremental" >
+        <PropertyGroup Condition="'$(MUXClassRegistrationsDir)' == ''">
+            <MUXClassRegistrationsDir>$(MSBuildThisFileDirectory)</MUXClassRegistrationsDir>
```

Comment:
consider renaming later, I changed all the "MUX" names I cloned form winui2 into more appropriate names.

Reply:
Actually, what does this do?  Does this apply to MRTCore?

Reply:
nevermind, this is a test app.